apps/site/src/app/admin/applicants/[uid]/components/HackerApplicantActions.tsx

```diff
@@ -0,0 +1,47 @@
+import { useContext, useState } from "react";
+
+import { Button, Input, SpaceBetween } from "@cloudscape-design/components";
+import { submitReview } from "@/app/admin/applicants/hackers/useApplicant";
+import { Uid } from "@/lib/userRecord";
+import UserContext from "@/lib/admin/UserContext";
+import { isReviewer } from "@/lib/admin/authorization";
+
+interface ApplicantActionsProps {
+	applicant: Uid;
+	submitReview: submitReview;
+}
+
+function HackerApplicantActions({
+	applicant,
+	submitReview,
+}: ApplicantActionsProps) {
+	const { roles } = useContext(UserContext);
+	const [value, setValue] = useState("");
+
+	if (!isReviewer(roles)) {
+		return null;
+	}
+
+	return (
+		<SpaceBetween direction="horizontal" size="xs">
+			<Input
+				onChange={({ detail }) => setValue(detail.value)}
+				value={value}
+				type="number"
+				inputMode="decimal"
+				placeholder="Applicant score"
+				step={0.1}
+			/>
```

Comment:
Note: according to the finalized rubric created by Applications, the only possible values for scores are 0, 0.5, 1, 1.5, 2, and so on, so changing the `step` value to `0.5` would make more sense